pages/app/index.tsx

```diff
@@ -40,6 +40,8 @@ function isAppLayoutPage(pageId?: string) {
     'container/sticky-permutations',
     'copy-to-clipboard/scenario-split-panel',
     'prompt-input/simple',
+    'funnel-analytics/static-single-page-flow',
+    'funnel-analytics/static-multi-page-flow',
```

Comment:
I recommend calling such pages as `with-app-layout` so it matches the existing pattern instead of constantly extending this (I would have posted the same comment on all exceptions above, if I would have a chance to review them).

Reply:
Will follow up with a rename.